.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Here is no need for such files from .vscode directory at requests. Please, read about .gitignore files=)